installer/bootstrap.py

```diff
@@ -1264,7 +1264,7 @@ def run():
         "--python",
         help="Install this minor version of Python (default: %(default)s)",
         default="3.9",
-        choices=("3.7", "3.8", "3.9"),
+        choices=("3.8", "3.9"),
```

Comment:
There a reason we do not allow 3.10 yet?

Reply:
Lots of packages don't build on 3.10 yet, but yes we can probably enable this in bootstrap

Reply:
I've added 3.10 in this PR. The step currently holding this back is our numpy incompatibility due to https://github.com/cctbx/cctbx_project/issues/627. Other packages that are not yet on 3.10 include eg. bitshuffle (https://github.com/kiyo-masui/bitshuffle/issues/94, https://github.com/conda-forge/bitshuffle-feedstock/pull/22), so you may still want to hold off for a bit.